### PR TITLE
Debris ActivityState

### PR DESCRIPTION
**Pull request by FG-TUM**
- New enum for signalling the activity state
 - Check in Accumulator to only apply perturbations to non-preserving particles
 - Tests

### CODE REVIEW

File: src/satellitePropagator/debris/Debris.h [hunk 1]
```diff
@@ -48,6 +48,27 @@ class Debris {
...
+        passive,
+
+        /**
+         * Can actively change its trajectory to avoid collisions.
+         */
+        evasive,
+
+        /**
+         * Same as evasive but additionally can actively maneuver to preserve their orbit.
+         * This is modelled only only applying Keplerian forces.
```
Comment (gomezzz) [inline src/satellitePropagator/debris/Debris.h:67]:
```suggestion
         * This is modelled by only applying Keplerian forces.
```

File: src/satellitePropagator/io/FileOutput.h [hunk 2]
```diff
@@ -282,7 +282,7 @@ void FileOutput<Container>::writeDebrisCSV(double t)
 }
 
 template <class Container>
-void FileOutput<Container>::writeDebrisTXT(double t)
+void FileOutput<Container>::writeDebrisTXT(double)
```
Comment (gomezzz) [inline src/satellitePropagator/io/FileOutput.h:285]: was this intentional? What does it actually do 🤔

File: src/satellitePropagator/physics/AccelerationAccumulator.h [hunk 3]
```diff
@@ -245,83 +245,86 @@ void AccelerationAccumulator<Container>::applyComponents() const
...
+                        new_acc_total[1] += new_acc_component[1];
+                        new_acc_total[2] += new_acc_component[2];
+                    }
                 }
-            }
-            if (config[SOL]) {
-                new_acc_component = SolComponent::apply(d, d_srp, sun_params);
-                if (write) {
-                    file_output->writeAcc_value(new_acc_component);
+                if (config[SOL]) {
```
Comment (gomezzz) [inline src/satellitePropagator/physics/AccelerationAccumulator.h:292]: The changelog seems to have gone a little haywire in this section, I'll nothing actually changed :D

**Review by gomezzz — APPROVED** [review]
Only minor changes

File: src/satellitePropagator/io/FileOutput.h (hunk 2 above)
Comment (FG-TUM) [reply src/satellitePropagator/io/FileOutput.h:285]: Yes this was intentional. By removing the name of the argument it becomes inaccessible, we signal that we will not use that parameter, and the warning that it is not used is removed. At the same time, the signature of the function stays intact.

File: src/satellitePropagator/physics/AccelerationAccumulator.h (hunk 3 above)
Comment (FG-TUM) [reply src/satellitePropagator/physics/AccelerationAccumulator.h:292]: If you [hide changes to whitespaces](https://github.blog/2018-05-01-ignore-white-space-in-code-review/) you see that there didn't actually happen anything ;)

**Pull request merged**